micrometer-spring-legacy/src/test/java/io/micrometer/spring/web/servlet/WebMvcMetricsIntegrationTest.java

```diff
@@ -92,7 +92,7 @@ public void rethrownExceptionIsRecordedInMetricTag() throws Exception {
 
         clock.add(SimpleConfig.DEFAULT_STEP);
         assertThat(this.registry.find("http.server.requests")
-            .tags("exception", "Exception2").value(Statistic.Count, 1.0).timer())
+            .tags("exception", "Exception2", "status", "500").value(Statistic.Count, 1.0).timer())
```

Comment:
This actually has a tag of `status=200`

Reply:
Exactly. That's the bug I wanted to demonstrate.